website/docs/explorer/querying.mdx

```diff
@@ -59,84 +59,23 @@ The schema looks like
 |------------| ---------| ----------------|---------|----------|------------------------|
 |`default/progress-delivery-demo2-32` | `flux-system`   | `HelmRelease` | `podinfo` | `Success` | `Release reconciliation succeeded`  |
 
-And represented in the UI like
```

Comment:
it seems that we miss a new header between schema and showing filters (or some other content that links both together) 

Reading it as currently is feels disconnected 

![Screenshot 2023-05-31 at 10 25 33](https://github.com/weaveworks/weave-gitops/assets/12957664/32347896-ba84-4906-9fd5-1ed1366b5007)